tests/test_env_default.py

```diff
@@ -0,0 +1,35 @@
+from marabunta.config import EnvDefault, BoolEnvDefault
```

Comment:
can you add a test on migration runner?
You should make sure that if we have:

1. flag false, it breaks
2. flag true execute more then one not installed step at once

see https://github.com/camptocamp/marabunta/blob/master/tests/test_migration_file.py